src/docs/templates/README.md

```diff
@@ -9,13 +9,13 @@ More information about `dotnet new` can be found at <https://docs.microsoft.com/
 Once the .NET Core SDK has been installed, type the following command to install the templates for creating Orchard Core web applications:
 
 ```CMD
-dotnet new -i OrchardCore.ProjectTemplates::1.0.0-beta3-*
+dotnet new -i OrchardCore.ProjectTemplates::1.0.0-rc1-*
```

Comment:
if we don't specify the version like `-i OrchardCore.ProjectTemplates`, would it just pick up the latest available?

Reply:
/cc @jtkech

Reply:
It only works with "stable" versions, and because we only have preview ones (beta) it doesn't work. It works also without the `*` but in this case it tool 1.0.0-beta3-71075 and not 1.0.0-beta3-71077 which both are on NuGet.

Reply:
So this string is the good one to use